code/modules/clothing/glasses/glasses.dm

```diff
@@ -7,14 +7,14 @@
 	species_fit = list(VOX_SHAPED, GREY_SHAPED)
 
 /obj/item/clothing/glasses/science
-	name = "Science Goggles"
+	name = "science goggles"
```

Comment:
Item names shouldn't be capitalized, not to mention that without `\improper` it's seen as a proper noun.